lib/ddg-atbgrabber.js

```diff
@@ -30,6 +29,7 @@ exports.install = function () {
           // set the ATB param from the loaded site if it was not set yet
           if (atbParam && !ss.storage.atb_set) {
             ss.storage.atb = atbParam;
+            ss.storage.set_atb = atbParam;
             ss.storage.atb_set = true;
```

Comment:
This is the only other reference to it, which should still work the same whether it's a boolean or a string: https://github.com/duckduckgo/firefox-zeroclickinfo/blob/master/lib/main.js#L212